scripts/list-contributors.js

```diff
@@ -21,24 +21,24 @@ module.exports = async function(callback) {
 
   try {
     const contributors = await kredits.Contributor.all()
+
+    contributors.forEach((c) => {
+      table.push([
+        c.id.toString(),
+        c.account,
+        `${c.name}`,
+        c.isCore,
+        ethers.utils.formatEther(c.balance),
+        c.totalKreditsEarned.toString(),
+        c.contributionsCount.toString(),
+        c.ipfsHash
+      ])
+    })
   } catch(e) {
     callback(e);
     return;
   }
 
-  contributors.forEach((c) => {
-    table.push([
-      c.id.toString(),
-      c.account,
-      `${c.name}`,
-      c.isCore,
-      ethers.utils.formatEther(c.balance),
-      c.totalKreditsEarned.toString(),
-      c.contributionsCount.toString(),
-      c.ipfsHash
-    ])
-  })
-
   console.log(table.toString())
```

Comment:
can we add this also in the` try {} catch` block? 
if something fails in this line we would also not see it.

Reply:
Feel free to change it.

Reply:
fyi: I generally don't like the need for those try/catch blocks everywhere... need to fix the root of the problem and it must be fixed on the truffle level or (as I mentioned what I am working on) that we don't rely on truffle again..